### DISCUSSION TITLE
Form resubmit does nothing.

### DISCUSSION THREAD

**Discussion by onweru** [DioxusLabs/dioxus#1802] [Q&A]
I'm working on a dioxus site. I have a login form that collects a username and password and sends a post request. 

If the values are wrong the backend obviously returns an error. In this case I show an error to the user asking them to check their details. The problem is that unless the user reloads the page, the submit button does nothing.

Has anyone come across this problem? Perhaps I'm missing something.

**Comment by ealmloff** [comment]
Dioxus prevents forms from submitting by default (https://github.com/DioxusLabs/dioxus/pull/926) if you have an event listener attached to form. If adding prevent default doesn't fix your issue, could you provide more information about your project: If possible the code, the renderer you are using, and if you are using client side routing

**Reply by onweru** [reply]
Thank you, l'll try what you have suggested. Will need time to confirm. 

I'm using the web renderer together with dioxus-router.

<img width="358" alt="SCR-20240110-nbop" src="https://github.com/DioxusLabs/dioxus/assets/16350351/042f1e4f-697d-4bd0-8812-f7c4a7e1b13f">

**Reply by onweru** [reply]
Update, adding prevent default helped, but you have to click 3 times to get the button working again.

**Reply by onweru** [reply]
Update, I fixed the said issue by adding dependencies to `use_future` ... works with `use_effect` too.
